docs/admin/postgres12_end_of_life_notice.mdx

```diff
@@ -0,0 +1,45 @@
+# Postgres 12 End of Life
+
+Sourcegraph's required minimum version of Postgres will be updated to Postgres 16 with upcoming releases, Sourcegraph 5.10. This change in required minimum version is a result of Postgres 12, the previous required minimum version, reaching end of life. This notification is coming to you because you are running a self-hosted instance of Sourcegraph. Please review the below information and reach out if you have any questions or would like to review the impact this change will have on you!
```

Comment:
We might want to take this out as it will be in the docs and not going out to customers: 
"This notification is coming to you because you are running a self-hosted instance of Sourcegraph. "